### DISCUSSION TITLE
No longer getting the screen overlay

### DISCUSSION THREAD

**Discussion by nafg** [focustimerhq/FocusTimer#655] [Q&A]
Hi, I'm excited to start using this, but I have a question. Initially I was getting the screen overlay but now I'm not, after I changed some settings.

Also, I'm confused about the difference and relationship between "Screen notifications" on the main Preferences page, and the Notifications option on the Plugins page.

**Comment by nafg** [comment]
Oh wait, they seem to be mutually exclusive sort of? Enabling one seems to disable the other, although sometimes I have to reopen Preferences to observe that. So I'm guessing Screen notification means the full-screen overlay and the Notifications plugin, whose description says "Pop-Up notifications," switches to the traditional desktop notifications instead?

**Reply by kamilprusko** [reply]
Hi @nafg :)

You're right. The "Notifications" should be removed from the plugins section - it doesn't do anything and it's confusing. I believe it's a remaining after an alternative implementation based on libnotify... I'll remove it in next release.

When "Screen notifications" is on, it may still popup regular notifications when it detects there is a fullscreen video player, or when when something is blocking the overlay from becoming a modal dialog.

**Reply by kamilprusko** [reply]
Removed in b5020cfc65ad1d17382691523abd5ef59dd7bfac
